gustaf/show.py

```diff
@@ -161,8 +161,8 @@ def cam_tuple_to_list(dict_cam):
         list_of_showables = []
         for sl in show_list:
             if not isinstance(sl, list):
-                sl = [sl]
-            for k, item in enumerate(sl):
+                sl = [sl]  # noqa: PLW2901
```

Comment:
Otherwise error that the iterator variable is overwritten inside the loop. Here it makes sense.

Reply:
I guess I don't even use `k` there, right?